zubhub_backend/zubhub/notifications/tests/test_views.py

```diff
@@ -0,0 +1,126 @@
+from django.urls import reverse, reverse_lazy
+from rest_framework.test import APIClient
+from django.test import TestCase
+from notifications.models import Notification
+from creators.models import Creator
+from notifications.serializers import NotificationSerializer
+from rest_framework import status
+from django.utils import timezone
+import json
+from django.test import Client
+from rest_framework.authtoken.models import Token
+from rest_framework import status
+from django.contrib.auth import get_user_model
+
+creator = get_user_model()
+
+class UserNotificationsAPIViewTest(TestCase):
+
+    @classmethod
+    def setUpTestData(self):
+        self.first_creator = Creator.objects.create(username="krishna")
+        self.second_creator = Creator.objects.create(username="kali")
+        self.client = APIClient()
+
+        self.payload = Notification.objects.create(
+            link="https://zubhub.com/exampleclap",
+            type=Notification.Type.CLAP,
+            recipient=self.first_creator,
+            message="Clap",
+            viewed=False
+        )
+        self.payload.save()
+        self.payload.sources.add(self.second_creator, self.second_creator)
+        self.payload.save()
+
+        self.auth = {
+            "username": "test_dummy2",
+            "email": "test_dummy2@gmail.com",
+            "password1": "qwertyu12345",
+            "password2": "qwertyu12345",
+            "phone": "+57778899066",
+            "dateOfBirth": "1998-02-20",
+            "location": "Nigeria",
+            "bio": "ddd",
+            "subscribe": True,
+        }
+
+    def test_get_user_notifications(self):
+        rgst = self.client.post(reverse("creators:signup_creator"), self.auth, format="json")
+        res = self.client.get(reverse('notifications:user-notifications'))
```

Comment:
Are there any steps required before completing registration (like email verification)

Reply:
@chaitanya-cloudknox I just checked and once you sign-up a new user it directly logs you in without any necessity of email-verification